packages/cli/src/controllers/binary-data.controller.ts

```diff
@@ -38,7 +38,14 @@ export class BinaryDataController {
 				} catch {}
 			}
 
-			if (mimeType) res.setHeader('Content-Type', mimeType);
+			if (mimeType) {
+				res.setHeader('Content-Type', mimeType);
+
+				// Sandbox html files when viewed in a browser
+				if (mimeType === 'text/html' && action === 'view') {
```

Comment:
Any chance that we could also have the old application/xhtml+xml mime type here ?

Reply:
How about 
```suggestion
				if (mimeType.includes('html') && action === 'view') {
```

Reply:
Looks good !